rust/src/longest_common_prefix.rs

```diff
@@ -0,0 +1,24 @@
+use crate::Solution;
+
+impl Solution {
+    pub fn longest_common_prefix(strs: Vec<String>) -> String {
+        if strs.len() == 0 {
+            return String::from("");
+        }
+        if strs.len() == 1 {
+            return strs[0].to_string();
```

Comment:
I would suggest to use `clone` instead of `to_string` here. Even if it does exactly the same thing it expresses much better what and why are you doing here